src/integrations/FacebookPixel/browser.js

```diff
@@ -390,16 +399,30 @@ class FacebookPixel {
         }
       }, legacyTo);
     } else if (event === "Products Searched") {
+      const contentIds = [];
+      const contents = [];
+
+      if (prodId) {
+        contentIds.push(prodId);
```

Comment:
check what if undefined goes here, because product searched does not have this field generally.

Reply:
if(prodId) check for null/undefined values.